tests/commands/test-kob-help.sh

```diff
@@ -1,21 +1,57 @@
 #!/bin/bash
-# check if kob is installed
-# Remove any old output.txt files  
-#rm output.txt
-# run the command
-kob help > output.txt
-# check the output of the command
-# if ( cat output.txt | grep "known pattern") 
- echo " Success"
-#  return 
- else 
-  echo "Failed $0"
-# check and remove
-rm output.txt
-
-
 
+function __test_kob_init
+{
+	commands="install uninstall status list version help update upgrade"
+    if [[ -d $KOBMAN_DIR ]]; then
+        source $KOBMAN_DIR/src/kobman-utils.sh
+        __kobman_echo_no_colour "kob found"
+        source $KOBMAN_DIR/bin/kobman-init.sh
+    else
+        echo "kob not found"
+        echo "Please install KOBman and try again"
+        echo "Exiting!"
+        exit 1
+    fi
+    
+}
 
+function __test_kob_execute
+{
+    kob help >> output.txt
+}
 
+function __test_kob_validate
+{
+    __kobman_echo_no_colour "Validating help command..." 
+    
+    for command in $commands;
```

Comment:
Find a way to compare template code rather than a loop, will save time and test will fail faster.